proto/storage/notification_schedule.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+message NotificationSchedule {
+    reserved 1; // id, future field
+    google.protobuf.Timestamp last_run = 2;
```

Comment:
From @dhaus67 in https://github.com/stackrox/stackrox/pull/4969

> ~~Should this not have an ID associated with it?~~
> 
> Reading through the code, I realized that only one notification schedule exists currently.
> However, looking at the way forward, it may be that you can group certain items (either types of tokens, or a specific group of tokens) to a notification schedule.
> So, maybe its worthwhile to add an ID field here (and also give it the 1 field tag), and have a static ID defined for the "default" notification schedule, similar to what we do for other default resources (e.g. access scopes)?